scripts/run_infrastructure.py

```diff
@@ -10,11 +10,11 @@
 
 DOCKER_WAIT = 5
 DOCKERFILE_DATASTORES = [
-    "mssql",
     "postgres",
     "mysql",
     "mongodb",
     "mariadb",
+    "mssql",
```

Comment:
just curious, did moving `mssql` last change / fix anything?

Reply:
in my mind: yes
objectively: hard to say